pom.xml

```diff
@@ -50,21 +50,23 @@
         <project.build.sourceEncoding>${encoding}</project.build.sourceEncoding>
         <project.reporting.outputEncoding>${encoding}</project.reporting.outputEncoding>
 
-        <sonarqube.version>9.4.0.54424</sonarqube.version>
-
         <sonar.organization>green-code-initiative</sonar.organization>
         <sonar.host.url>https://sonarcloud.io</sonar.host.url>
 
-        <sonarjava.version>7.15.0.30507</sonarjava.version>
-        <sonarpython.version>3.19.0.10254</sonarpython.version>
-        <sonarphp.version>3.25.0.9077</sonarphp.version>
-        <sonarjavascript.version>9.13.0.20537</sonarjavascript.version>
+        <sonarqube.version>9.4.0.54424</sonarqube.version>
+        <sonar-plugin-api-impl.version>10.0.0.68432</sonar-plugin-api-impl.version>
+        <sonarjava.version>7.19.0.31550</sonarjava.version>
+        <sonarpython.version>4.3.0.11660</sonarpython.version>
+        <sonarphp.version>3.29.0.9684</sonarphp.version>
+        <sonarjavascript.version>10.2.0.21568</sonarjavascript.version>
+
+        <sonar-analyzer-commons.version>2.5.0.1358</sonar-analyzer-commons.version>
 
-        <sonar-plugin-api-impl.version>9.7.1.62043</sonar-plugin-api-impl.version>
-        <sonar-analyzer-commons.version>2.1.0.1111</sonar-analyzer-commons.version>
         <sonar-packaging.version>1.21.0.505</sonar-packaging.version>
         <sonar.skipDependenciesPackaging>true</sonar.skipDependenciesPackaging>
 
+
+        <jacoco.version>0.8.10</jacoco.version>
```

Comment:
I have the impression that this variable is not used?
Further down, the jacoco plugin always uses a hard-coded version.

Reply:
you're rigth ! I correct it.